autopush/router/webpush.py

```diff
@@ -132,6 +132,11 @@ def _save_notification(self, uaid, notification, month_table):
                                   headers={"TTL": str(notification.ttl),
                                            "Location": location},
                                   logged_status=204)
+        self.log.info("Saving message data",
```

Comment:
We already log whether it was stored or online delivery in web/webpush.py, shouldn't the size and such be added there?